### PR TITLE
#232 solved lint issues

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <subhamkumarrai03@gmail.com>

### CODE REVIEW

**Comment by leecalcote** [comment]
Closes #232

File: mesheryctl/cmd/vars.go [hunk 1]
```diff
@@ -24,14 +24,14 @@ import (
...
+	fileURL                = "https://raw.githubusercontent.com/layer5io/meshery/master/docker-compose.yaml"
 	mesheryLocalFolder     = ".meshery"
 	dockerComposeFile      = mesheryLocalFolder + "/meshery.yaml"
-	dockerComposeBinaryUrl = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
+	dockerComposeBinaryURL = "https://github.com/docker/compose/releases/download/1.24.1/docker-compose"
 	dockerComposeBinary    = "/usr/local/bin/docker-compose"
 )
 
-func DownloadFile(filepath string, url string) error {
+func downloadFile(filepath string, url string) error {
```
Comment (leecalcote) [inline mesheryctl/cmd/vars.go]: @girishranganathan does this change actually address the lint issue: ```/go/src/github.com/layer5io/l5act/mesheryctl/cmd/vars.go:34:1: exported function DownloadFile should have comment or be unexported```?

Comment (girishranganathan) [reply mesheryctl/cmd/vars.go]: 😁 yes. Only exported methods need a comment.

Comment (subhamkrai) [reply mesheryctl/cmd/vars.go]: need comment or we can simple change it to lowercase both will work fine

Comment (leecalcote) [reply mesheryctl/cmd/vars.go]: Oh, I see. Very good!

**Pull request merged**